nemo_skills/prompt/template/llama3-base-fewshot.yaml

```diff
@@ -0,0 +1,31 @@
+# Prompt specification for Llama3-base model
```

Comment:
Why do we need this? This should be done as part of few shot examples in the config and not a separate template